admin_site/system/views.py

```diff
@@ -1640,7 +1649,7 @@ def get_queryset(self):
 
         try:
             limit = int(params.get('limit', 20))
-        except:
+        except e:
```

Comment:
This will catch and drop _all_ exceptions, including a bunch of system-internal stuff like `MemoryError` and `KeyboardInterrupt`. What you want here is probably `except ValueError` (and you don't need to capture it as `e`, since you don't do anything with that `e`).

(Also, er, you have a default of 20 in the call to `params.get` and a default of 10 in the error case. Shouldn't it be one or the other?)

Reply:
Adding the 'e' was a part of the PEP-8 formatting I impulsively made. I see how formatting other code than specifically related to current feature implementation is a no-no from now on. I apologize for the confusion.

Reply:
I'll changed the exception to _ValueError_ without assigning it, and the **pycodestyle**(pep8-validator tool) is OK with that. I guess you either need to specify the type of exception, or you assign it to a variable...?